src/elements/file-selector/file-selector.ts

```diff
@@ -1,21 +1,25 @@
-import type { CSSResultGroup, TemplateResult } from 'lit';
-import { LitElement, nothing } from 'lit';
-import { customElement, property, state } from 'lit/decorators.js';
+import { type CSSResultGroup, LitElement, nothing, type TemplateResult } from 'lit';
```

Comment:
I detected that setting name on file selector is not reflected in form data. But so far I don't know why.

```html
<form>
  <label for="fil">Hello</label>
  <sbb-file-selector
    id="fil"
    name="test"
    data-testid="sbb-file-selector"
    variant="default"
    size="m"
    title-content="Title"
    multiple-mode="default"
    accessibility-label="Select from hard disk"
  ></sbb-file-selector>
</form>
```


While $0 is the form:
```ts
for(const foo of (new FormData($0)).entries()) {console.log(foo)}
```

Reply:
To appear in the FormData the element needs to have a value (I guess it was empty in your test)

Reply:
hmm I added a file, which was in the form data, but the name was missing

Reply:
Maybe we can try together since I might doing something different

Reply:
Lets do it in the call